### ISSUE TITLE
Running multiple features with webpack bundler throws error

### ISSUE THREAD

**Issue by jp7677** [badeball/cypress-cucumber-preprocessor#694]
Hi there,

Running the cypress-cucumber-preprocessor with the webpack example provided here https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples/webpack works perfectly. Though as soon as you add a second feature file, e.g. by just making a copy of the `duckduckgo.feature` and  `duckduckgo.ts` files, running all integration tests fails with `Expected to find an error message (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)`. Note that running the features separately works fine. It is really just the run-all that fails.

The error as presented in Cypress is:
```
Expected to find an error message (this might be a bug, please report at https://github.com/badeball/cypress-cucumber-preprocessor)

Because this error occurred during a after each hook we are skipping all of the remaining tests. 
node_modules/@badeball/cypress-cucumber-preprocessor/lib/assertions.js:7:1
   5 | const package_json_1 = require("../package.json");
   6 | function fail(message) {
>  7 |     throw new Error(`${message} (this might be a bug, please report at ${package_json_1.homepage})`);
     | ^
   8 | }
   9 | exports.fail = fail;
  10 | function assert(value, message) {
```

### Current behavior

See this screenshot what happens when running all tests.

![Screenshot from 2022-04-13 14-03-10](https://user-images.githubusercontent.com/17219293/163175796-4e310a0c-0ec9-445f-8341-d522250cb8dd.png)

### Desired behavior

Running all tests should succeed with multiples feature files.

### Test code to reproduce

As stated above, just making a copy of the feature/ts files in https://github.com/badeball/cypress-cucumber-preprocessor/tree/master/examples/webpack is sufficient to reproduce this behavior. 

### Versions

* **Cypress version**: 9.5.4
* **Preprocessor version**: 9.0.0
* **Node version**: v16.14.2

**Comment by badeball** [comment]
Hey @jp7677, thanks for the feedback! I've released v9.0.1 that fixes this.

**Comment by jp7677** [comment]
Thanks a lot, that was really fast!
I'm curious what caused this, do you have a link to the commit?

**Comment by badeball** [comment]
Ouf, looks like I was a biit too quick there. v9.0.1 doesn't actually contain the fix, it's basically identical to v9.0.0. I've released v9.0.2 now, which contains 49e9723ae8c21a225962a64dcf588aa91bd6eec5.

**Comment by jp7677** [comment]
I can confirm that version 9.0.2 does indeed fixes this, thanks a lot for the very fast help!
